support/values.yaml

```diff
@@ -71,3 +71,26 @@ grafana:
           access: proxy
           isDefault: true
           editable: false
+
+# Values reference for nfs-server-provisioner
+# https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/blob/master/deploy/helm/values.yaml
+nfs-server-provisioner:
+  enabled: false
+  storageClass:
+    # Users will have to opt-in to using NFS-type storage under
+    # singleuser.storage.dynamic.storageClass
+    name: nfs
+    create: true
+    defaultClass: false
+    mountOptions:
+      - vers=4.2
+      - soft
+      - noatime
```

Comment:
@yuvipanda the default options in the nfs-server-provisioner helm chart is `- vers=3`.

Would it make sense to create a PR suggesting a new default in the helm chart perhaps? I know too little to say that it does indeed make sense or not.

This is the upstream repo to update if so: https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner

Reply:
Yeah, I think the default should be 4.2 since nfs-ganesha supports it, unless they have a strong reason to not do that. Would be nice to open a PR upstream!

Reply:
Opened https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/issues/64